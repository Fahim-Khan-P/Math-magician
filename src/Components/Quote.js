import React from 'react';
import '../App.css';

const Quote = () => (
  <div className="qouteInformation">
    <h2>This is the Quote page of our App!</h2>
    <p className="qoute">
      Lorem ipsum dolor sit amet,
      consectetur adipiscing elit,
      sed do eiusmod tempor incid
      ut labore et dolore magna aliqua.
      Ut enim ad minim veniam,
      quis nostrud exercitation ullamco laboris nisi ut aliqu
      ex ea commodo consequat. Duis aute irure dolor in repreh
      eit dolore eu fugiat nulla pariatur. Excepteur sint occ
      cupidatat non proident, sunt in culpa qui officia deserunt
      mollit anim id est laborum.
      Lorem ipsum dolor sit amet,
      consectetur adipiscing elit,
      sed do eiusmod tempor incid
      ut labore et dolore magna aliqua.
      Ut enim ad minim veniam,
      quis nostrud exercitation ullamco laboris nisi ut aliqu
      ex ea commodo consequat. Duis aute irure dolor in repreh
      eit dolore eu fugiat nulla pariatur. Excepteur sint occ
      cupidatat non proident, sunt in culpa qui officia deserunt
      mollit anim id est laborum.
      Lorem ipsum dolor sit amet,
      consectetur adipiscing elit,
      sed do eiusmod tempor incid
    </p>
  </div>
);

export default Quote;
