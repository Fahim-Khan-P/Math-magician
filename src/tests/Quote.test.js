import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Components/Quote';

test('should render quotes', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
