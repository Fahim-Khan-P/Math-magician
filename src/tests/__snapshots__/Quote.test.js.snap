// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render quotes 1`] = `
<div
  className="qouteInformation"
>
  <p
    className="qoute"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incid ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliqu ex ea commodo consequat. Duis aute irure dolor in repreh eit dolore eu fugiat nulla pariatur. Excepteur sint occ cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incid ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliqu ex ea commodo consequat. Duis aute irure dolor in repreh eit dolore eu fugiat nulla pariatur. Excepteur sint occ cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incid
  </p>
</div>
`;
